IFComp/t/entry_processing.t

```diff
@@ -0,0 +1,120 @@
+use strict;
+use warnings;
+use Test::More;
+
+use FindBin;
+use lib ("$FindBin::Bin/lib");
+use IFCompTest;
+my $schema = IFCompTest->init_schema();
+
+my $entry_directory =
+    Path::Class::Dir->new("$FindBin::Bin/platform_test_entries");
+
+$schema->entry_directory($entry_directory);
+
```

Comment:
It probably made sense years ago. Maybe.

I'll see if things blow up if I just move them into the test lib, and if not I'll check that in.